toolkit/scalebar/src/main/java/com/arcgismaps/toolkit/scalebar/internal/ScalebarViewModel.kt

```diff
@@ -132,11 +129,11 @@ internal class ScalebarViewModel(
      *
      * @since 200.7.0
      */
-    fun updateScaleBar(
+    internal fun updateScaleBar(
```

Comment:
This method does not directly update the scalebar
```suggestion
    internal fun computeScalebarProperties(
```